include/mbgl/util/string_indexer.hpp

```diff
@@ -9,6 +9,8 @@
 #include <mutex>
 #include <shared_mutex>
 
+using namespace std::string_view_literals;
```

Comment:
Better not to have this in the global namespace, but at least within `mbgl` or not in the header at all.